.gitignore

```diff
@@ -74,3 +74,8 @@ paper.yml-README.txt
 .HARD_FAILED
 .NO_UNPATCH
 .NO_PULL
+
+plugins/spark/tmp-client/
+plugins/CustomDiscs/musicdata
+plugins/CustomDiscs/data.yml
+world/datapacks/copper-door-cutting/LICENSE
```

Comment:
NIT: This file won't be created automatically, so you could just delete it. But regardless, we should keep the license.

Reply:
Retracked custom discs data and copper door cutting license, which should solve this.